resources/activity.md

```diff
@@ -12,6 +12,22 @@ Default `orderBy`: `createdDateUTC DESC`
 |`/api/participants/{participantID}/activity`|`array`|To find `{participantID}`, visit your Participant's Fundraising Page. Look for the `&participantID=` URL parameter.|
 |`/api/teams/{teamID}/activity`|`array`|To find `{teamID}`, visit your Team's Fundraising Page. Look for the `&teamID=` URL parameter.|
 
+### URL Aliases
+
+You can use URL aliases instead of numeric IDs in any of these endpoints:
+
+```
+# Using numeric IDs
+GET /api/1.3/extra-life-1234/events/605/activity
```

Comment:
what does `extra-life-1234` signify here? also observing we could nudge the API version up to 1.6 (not sure there are perceptible changes, but it's the latest active version we support).